### ISSUE TITLE
Add ability to enable and disable model

### ISSUE THREAD

**Issue by wtlgo** [germaniumhq/mopyx#10]
Please, can you add an option to enable and disable models' ability to attach to renderers?

Right now it works like this:
```python
# The model
@model
def Model:
    def __init(self):
        self.foo = "bar"

model = Model()

#The render function
@render
def test_render():
    print("Re-render:", model.foo)

# Runs the renderer and starts listening to model.foo
test_render()

# Updates the model.foo and runs the renderer
model.foo = "baz"
```

This will print
```log
Re-render: bar
Re-render: baz
```

And here's what I mean:
```python
# Still the model but in inert state
@model(is_active=False)
def Model:
    def __init(self):
        self.foo = "bar"

model = Model()

# The same renderer
@render
def test_render():
    print("Re-render:", model.foo)

# Will run the renderer, but will not start to listen to model.foo. 
# Basically, at this point, the model behaves like a regular data class
test_render()

# Just updates the state
model.foo = "bar"

# Now model becomes active and can attach to renderers
activate(model)

# At this point the library works normally

model.foo = "qux"
test_render()

mode.foo = "quux"
```

And this will print out
```log
Re-render: bar
Re-render: qux
Re-render: quux
```
